include/pyre3.14t/audit.h

```
@@ -0,0 +1,53 @@
+ */
+#ifndef PYRE_AUDIT_H
+#define PYRE_AUDIT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* `sys_audit_tstate` — the arguments are whatever the format builds, wrapped
+   in a one-element tuple when the format builds a single value rather than a
+   tuple.  An empty or NULL format raises the event with no arguments. */
+static inline int PySys_Audit(const char *event, const char *argFormat, ...)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'PySys_Audit' of similar type ('const char *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)